OrbitGl/App.cpp

```diff
@@ -193,6 +194,8 @@ void OrbitApp::PostInit() {
     // TODO: Replace refresh_timeout with config option. Let users to modify it.
     process_manager_ = ProcessManager::Create(grpc_channel_, absl::Milliseconds(1000));
 
+    tracepoint_manager_ = TracepointServiceClient::Create(grpc_channel_);
```

Comment:
I'd move this down to line ~245 to all the other stuff with the tracepoint_manager_ such that it is closer together.

It might be a good idea to guard this and the asynchronous fetching of the tracepoint by the flag. This only should happen once the feature is activated.